### ISSUE TITLE
Kitcoin should be translated into other languages

### ISSUE THREAD

**Issue by edcous** [codeths/ethsbell-rewrite#97]
**Describe what's wrong**
Kitcoin is currently in English only. We should have options for other languages to ensure accessibility.
**Describe what you'd like to change**
The extension especially and the website should have some sort of language change option.
